config/exchanges.json

```diff
@@ -972,8 +976,8 @@
   },
   "DK-DK1->NO-NO2": {
     "capacity": [
-      -1700,
-      1700
+      -715,
```

Comment:
I think this value is too low. The exchange values were above 1000 MW in both directions today.

Reply:
Nice catch!
The map I was using is a bit messy so probably just picked the wrong number...
![image](https://user-images.githubusercontent.com/30777521/167892934-984393f6-6ba0-438d-840b-ca7935aa586d.png)

I'll update it right away.